blog/2023-01-10-first-offsite/index.mdx

```diff
@@ -0,0 +1,123 @@
+---
+slug: first-offsite-retro
+title: Lessons from Firezone’s first offsite
+authors: [jason]
+tags: [company, offsite]
+---
+
+My first startup job was at [Strong](https://www.strong.app/), a workout journal app that was pretty popular at the time. The founders were “digital nomads,” and the team was fully remote.
+
+Back in 2016, this was highly uncommon.
+
+We spent a lot of time creating processes to work together effectively on opposite sides of the world. One of these rituals was regularly meeting up in person.
+
+![digital nomads](https://user-images.githubusercontent.com/52545545/211626078-6f64a4f4-ec37-40cd-8bb5-a6b9b122f351.png)
+
+At first, it was by accident. Since everyone was traveling anyway, we would coordinate being in the same city at the same time.
+
+After working in person we quickly noticed, the way we worked remotely improved. People were more comfortable sharing their ideas, better at giving constructive criticism, and happier in general.
+
+It felt better to work together.
+
+## Why try this at Firezone?
+
+If you believe a small network security startup can take a sizable bite out of the usual suspects of legacy VPN products, there are a few ways it can happen. Many involve assembling an all-star team who can execute quickly to build a great product.
+
+This is much easier when everyone is comfortable sharing ideas and engaging in lively debate. Plus, it’s a lot more fun.
+
+So, we decided to try it out. After much planning, canceled bookings, and a few long flights later, our team met up in Santa Rosa, California.
+
+![offsite office](https://user-images.githubusercontent.com/52545545/211517211-ff1d5503-a367-4e6b-a670-067c447ed415.png)
+
+:::info
+My co-founder Jamil shares similar views about working together in person. He’s an even earlier pioneer in working remotely. I keep bugging him to write a blog about his 3 year journey living in a tiny house towed by his truck.
+
+For now, I’ll leave you with a link to his [slideshow](https://tinyhouse.jamilbk.com/).
+:::
+
+## What did we work on?
+
+Our itinerary included architecture planning, a multi-day hackathon, and plenty of fun activities. Let’s start with what we built.
+
+### REST API v0
+
+One common compliment from our users is how snappy and effortless it is to manage Firezone with the Web UI. Many IT teams support hundreds of concurrent users without a hitch. However, as fetching as our web portal is, we’d prefer if you rarely had to open it up. 
+
+Our REST API follows a series of updates to make Firezone easier to automate. This is why we made Docker the preferred (and soon-to-be only) method of deploying Firezone. And also why we’re building on the work from our community to release a first-party Terraform provider.
```

Comment:
```suggestion
Our REST API follows a series of updates to make Firezone easier to automate. This is why we made Docker the preferred method of deploying Firezone. And also why we’re building on the work from our community to release a first-party Terraform provider.
```

Will announce the details in 0.7 blogpost, can link to that from here when ready.